numpy/_core/tests/test_mem_policy.py

```diff
@@ -290,30 +290,32 @@ class MyArr(np.ndarray):
 
 
 async def concurrent_context1(get_module, orig_policy_name, event):
+    from numpy._core.multiarray import get_handler_name
     if orig_policy_name == 'default_allocator':
         get_module.set_secret_data_policy()
-        assert np.core.multiarray.get_handler_name() == 'secret_data_allocator'
+        assert get_handler_name() == 'secret_data_allocator'
     else:
         get_module.set_old_policy(None)
-        assert np.core.multiarray.get_handler_name() == 'default_allocator'
+        assert get_handler_name() == 'default_allocator'
     event.set()
 
 
 async def concurrent_context2(get_module, orig_policy_name, event):
+    from numpy._core.multiarray import get_handler_name
```

Comment:
If you want to make this simpler I don't think there's any problem with importing `numpy._core.multiarray` at top-level in this file. 

Please also keep how all of these tests are structured consistent, if you want to remove all the explicit `numpy._core.multiarray` do that in every test, not just some.

Reply:
Sure! I moved `numpy._core.multiarray` to a top-level import in files where it was imported more than once in unit tests.